docker/production.yml

```diff
@@ -31,6 +31,7 @@ services:
         - POSTS_URL=${POSTS_URL}
         - SEARCH_URL=${SEARCH_URL}
         - FEED_DISCOVERY_URL=${FEED_DISCOVERY_URL}
+        - STATUS_URL=${STATUS_URL}
```

Comment:
I'm confused about whether or not we still need this, if he have it above for the nginx build of the frontend?  cc @manekenpix

Reply:
I don't think we need it here(or anything related to the telescope container).

Reply:
@HyperTHD it looks like you haven't fixed @manekenpix's comment yet.

NOTE: our frontend is built here now: https://github.com/Seneca-CDOT/telescope/blob/master/docker/docker-compose.yml#L269-L293, which you've done above, so I think this can be removed.  The `production.yml` file shouldn't need to be changed.